docs/products/kafka/karapace/howto/manage-schema-registry-authorization.rst

```diff
@@ -0,0 +1,33 @@
+Manage Karapace schema registry authorization
+=============================================
+
+Karapace schema registry authorization allows you to authenticate the user, to control access to individual `Karapace schema registry REST API endpoints <https://github.com/aiven/karapace>`_, and to filter the content the endpoints return.
+
+.. tip:: 
+  Some older Aiven for Apache Kafka® services may not have this feature enabled by default, and you need to :doc:`enable Karapace schema registry authorization <../howto/enable-schema-registry-authorization>`.
+
+Karapace schema registry authorization is configured using :doc:`Access Control Lists (ACLs) <../concepts/schema-registry-authorization>`. You can manage the Karapace schema registry authorization ACL entries using the :doc:`Aiven CLI </docs/tools/cli/service/schema-registry-acl>`. 
+
+Using the Aiven CLI commands, you can 
+
+* Add ACL
+* Delete ACL
+* View ACL list
+
+For more information on the ACL commands, the required parameters and examples, see :doc:`avn service schema-registry-acl </docs/tools/cli/service/schema-registry-acl>`.
+
+Manage resources via Terraform
```

Comment:
Should we show this as part of a [Terraform recipe](https://docs.aiven.io/docs/tools/terraform/reference/cookbook.html) and link from here?

Reply:
This is a good idea. I would take this up as the next iteration.